homeassistant/components/remote_calendar/config_flow.py

```diff
@@ -69,7 +70,9 @@ async def async_step_user(
                 )
             except CalendarParseError as err:
                 errors["base"] = "invalid_ics_file"
-                _LOGGER.debug("Invalid .ics file: %s", err)
+                msg = "There was a problem reading the calendar information:"
+                _LOGGER.error("%s %s", msg, err.message)
+                _LOGGER.debug("%s %s", msg, str(err.detailed_error))
```

Comment:
I'd just inline this:

```suggestion
                _LOGGER.error("Error reading the calendar information: %s", err.message)
                _LOGGER.debug("Additional calendar error detail: %s", msg, str(err.detailed_error))
```